app/timetables/tests/test_models.py

```diff
@@ -2,9 +2,10 @@
 
 from django.core.exceptions import ValidationError
 from django.test import TestCase
+from django.utils import timezone
 
 from app.timetables.models import (Weekday, Meal, MealOption, Course,
-                                   Timetable, Dish, MenuItem)
+                                   Timetable, Dish, MenuItem, Vendor)
```

Comment:
Same comment as above `from app.timetables import models as tt_models`. Something like this.